content/updates/2019-02-26-quarterly-update.md

```diff
@@ -66,7 +64,7 @@ Using a new CLI plugin, you can now mine a greater range of logs and metrics tha
 
 ### Ensure you have the latest CLI client!
 
-While you’re checking out that new log-cache plugin, now would be a good time to make sure you’ve installed the latest version of the Cloud Foundry CLI, version 6.42.0. While the CLI is generally forward-compatible as we update cloud.gov, you’ll only get new features and bugfixes if you [keep your CLI up to date](https://cloud.gov/docs/getting-started/setup/#set-up-the-command-line). For example, the most recent CLI version enables [space-to-space networking policies](https://github.com/cloudfoundry/cli/releases/tag/v6.42.0), a capability only previously available by working directly with the Cloud Foundry API.
+While you’re checking out that new log-cache plugin, now would be a good time to make sure you’ve installed the latest version of the Cloud Foundry CLI, version 6.43.0. While the CLI is generally forward-compatible as we update cloud.gov, you’ll only get new features and bugfixes if you [keep your CLI up to date](https://cloud.gov/docs/getting-started/setup/#set-up-the-command-line). For example, the most recent CLI version ensures your credentials are not exposed inadvertently when running with the verbose flag, while the [version before that](https://github.com/cloudfoundry/cli/releases/tag/v6.43.0) added the ability to manage [space-to-space networking policies](https://github.com/cloudfoundry/cli/releases/tag/v6.42.0), a capability only previously available by working directly with the Cloud Foundry API.
```

Comment:
"forward-compatible" -> "compatible with newer versions"

Reply:
Clarify "you’ll only get new features and bugfixes" -> you can only benefit from "some features and bugfixes"

Reply:
"For example, the most recent CLI version ensures your credentials are not exposed inadvertently" -> "For example, the most recent CLI version hides your credentials when running with the verbose flag, so it's easier to copy-and-paste your transcripts to other people." Split sentence at "while the" - start with "The version from that".